sdk/eventhub/README.md

```diff
@@ -1,8 +1,22 @@
-# Azure Event Hubs for node.js
+# Azure Event Hubs client libraries for JavaScript
 
-We have two client libraries for Azure Event Hubs
-- `@azure/event-hubs` - The Event Hubs client for sending and receiving messages. You can find more info over [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs).
-- `@azure/event-processor-host` - The Event Processor Host for efficient receiving of messages. You can find more info over [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-processor-host).
+[Azure Event Hubs](https://azure.microsoft.com/services/service-bus/) is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers. This lets you process and analyze the massive amounts of data produced by your connected devices and applications.
```

Comment:
```suggestion
[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers. This lets you process and analyze the massive amounts of data produced by your connected devices and applications.
```

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/12599/commits/c5b5e1855af0b909443da225354ea0eb8c3b6b0d